net/api/pb/threadsnet.proto

```diff
@@ -49,6 +49,7 @@ message LogInfo {
     bytes privKey = 3;
     repeated bytes addrs = 4;
     bytes head = 5;
+    bool direct = 6;
```

Comment:
Went with direct here, that way the zero value is false, making it backward compatible.

Reply:
Is this needed here? The value of direct should be different for different peers... maybe I'll see more about it below.